### PR TITLE
Add react sdk quickstart

### PR DESCRIPTION
**Pull request by mooreds**
TODO
* [ ] clean up old react apps
* [ ] point all react blog posts at this documentation
* [x] have someone work through it.

### CODE REVIEW

File: site/docs/v1/tech/tutorials/integrate-react.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+You can start up the server and visit the URL displayed to ensure the default application works.
+
+```shell
+npm run dev
+```
+
+You'll want to open another terminal window to continue. Edit `src/App.jsx` to make changes to the view to test out the automatic reloading.
+
+{% include _callout-tip.liquid content="If you are using a different port for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly." %}
```
Comment (vcampitelli) [inline site/docs/v1/tech/tutorials/integrate-react.md:93]: This is giving me an error when building.

> Liquid Exception: Invalid syntax for include tag: app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly." Valid syntax: {% include file.ext param='value' param2='value' %} in docs/v1/tech/tutorials/integrate-react.md

I'm using Jekyll 4.3.2 and Liquid 4.0.4. I had to change it to this to make it work.

```asciidoc
{% capture content %}If you are using a different port for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly.{% endcapture %}
{% include _callout-tip.liquid content=content %}
```

File: site/docs/v1/tech/tutorials/integrate-react.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
```
Comment (vcampitelli) [inline site/docs/v1/tech/tutorials/integrate-react.md:25]: This is not replacing `{language}`. In #2091, I've created a `site/_includes/docs/v1/tech/tutorials/_integrate-add-user.liquid` file to handle this. Maybe we should use it instead?

Also, we lost the cool `->` Unicode replacement now that we are using Markdown.

File: site/docs/v1/tech/tutorials/integrate-react.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+```shell
+npm run setup
+```
+
+If you want, you can http://localhost:9011[login to your instance] and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application without any framework, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
```
Comment (vcampitelli) [inline site/docs/v1/tech/tutorials/integrate-react.md:65]:
```suggestion
Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
```

**Review by vcampitelli — CHANGES_REQUESTED** [review]

File: site/docs/v1/tech/tutorials/integrate-react.md (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-react.md:93]: thanks!

File: site/docs/v1/tech/tutorials/integrate-react.md (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-react.md:25]: Thanks. I changed this one to use your implementation. We'll probably have to figure out a way to merge the include file stuff, I think I was following your lead.

For the unicode stuff, I just spend 15 minutes trying to figure it out and didn't see where we actually do the replacement. This is all going to be moved over to a new static site generator, so I think we should tackle that then.

File: site/docs/v1/tech/tutorials/integrate-react.md [hunk 4]
```diff
@@ -90,7 +90,8 @@ npm run dev
 
 You'll want to open another terminal window to continue. Edit `src/App.jsx` to make changes to the view to test out the automatic reloading.
 
-{% include _callout-tip.liquid content="If you are using a different port for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly." %}
+{% capture callout_content %}If you a different port than 5173 for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match.{% endcapture %}
```
Comment (vcampitelli) [inline site/docs/v1/tech/tutorials/integrate-react.md:93]:
```suggestion
{% capture callout_content %}If you are using a different port than 5173 for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match.{% endcapture %}
```

**Review by vcampitelli — CHANGES_REQUESTED** [review]

**Comment by mooreds** [comment]
Merging because there was only that one typo.

**Pull request merged**